engines/supernova/game-manager.cpp

```diff
@@ -525,13 +529,16 @@ void GameManager::processInput() {
 			case onInventoryArrowDown:
 				// Fallthrough
 				_guiInventoryArrow[_mouseField - 768].setHighlight(true);
+				_ttsMan->say(_guiInventoryArrow[_mouseField - 768].getText(), Common::TextToSpeechManager::INTERRUPT_NO_REPEAT, Common::kDos850);
```

Comment:
Don't forget to test that the `_ttsMan` is not a nullptr and that tts is enabled.

Reply:
Also see my comments in the pull request. Quick summary as a reminder:
 * You don't need to specify the `Common::TextToSpeechManager::INTERRUPT_NO_REPEAT` since this is the default.
 * I am not sur the inventory arrow text can actually be spoken.
 * For the case below I think you need to get the text from the `_currentInputObject` and not from the `_guiInventory` as the later may not be set.